src/py/mains/consensus.py

```diff
@@ -13,6 +13,27 @@
 falcon.free_consensus_data.argtypes = [ POINTER(falcon_kit.ConsensusData) ]
 
 
+def get_longest_reads(seqs, max_n_read, max_cov_aln, sort=True):
+    # including the sort kwarg allows us to avoid a redundant sort
+    # in get_consensus_trimmed()
+    if sort: 
+        seqs = seqs[:1] + sorted(seqs[1:], key=lambda x: -len(x))
+
+    longest_n_reads = max_n_read
+    if max_cov_aln > 0:
+        longest_n_reads = 0
```

Comment:
Hey @pb-cdunn,

Ah, right. I always forget that the `to` index in slice notation is 1-based...